content/spin/v2/contributing-docs.md

```diff
@@ -175,9 +175,9 @@ hello
 
 The no copy annotation (`<!-- @nocpy -->`) precedes a code block where no copy and pasting of code is intended. If using the no copy attribute please still be sure to add the appropriate syntax highlighting to your code block (for display purposes). For example:
 
-`<!-- @nocpy -->`
+<!-- @nocpy -->
```

Comment:
I believe this line is intended to show the use of the `@nocpy` directive, so it's correct that it's backticked.

Reply:
But then you can copy from the example box, no? Seems strange to have it that way.

Reply:
I guess we could do a backticked "nocpy" for display followed by an actual nocpy to suppress the copy button on the sample text?  I'm not too fussed about it but we do need to show what the contributor has to type as that is the whole point of the section _grin_

Reply:
Before: <img width="736" alt="Screenshot 2024-05-02 at 10 11 35 PM" src="https://github.com/fermyon/developer/assets/111464359/f64a8897-5bde-42ea-8523-0adf6b7a3ee9">
After: <img width="752" alt="Screenshot 2024-05-02 at 10 11 07 PM" src="https://github.com/fermyon/developer/assets/111464359/d950134b-5ea1-4c04-b73e-088584501a91">

Reply:
@garikAsplund Yes. It is no longer showing the `<!-- nocpy -->` directive.  _The purpose of the example is the show the usage of the directive._  Please do not make it invisible.

Reply:
I think the directive is still there though--just in the paragraph--very much like in the previous selective copy example. The example _should be_ the code block. Note the icon difference between the two. It's strange to say "here's an example of a no copy block" and then be able to copy it.

Reply:
I understand that.  But the purpose of the example is to show _the source that the contributor would type, including the directive._

Perhaps an alternative way to approach this would be to code-quote the "Some generic code..." as well as the directive, so that it is all shown as "source code the contributor would type"?  Or to do that followed by the render with just the uncopyable code block a la:

> For example, if your Markdown file contains:
>
> _code block with directive and example_
>
> Then it will be rendered as:
>
> _the uncopyable block_

Reply:
I think that would be a little clearer than present and hopefully more consistent if the selective copy followed that format too.

Reply:
Don't mean to complicate or nitpick--I tried adding in the code block as an example and wasn't happy with it because rendering a to-be-copied code block _within_ a code block is insanely hard! `<code>` wasn't good, `` \` `` wasn't attractive, ZWJ has a good look but messes with user functionality in a hidden way, and on and on.

I still question whether there's a need for `<!-- @nocpy -->` as a standalone line especially if the example above doesn't have it.

<img width="927" alt="Screenshot 2024-05-05 at 10 45 23 AM" src="https://github.com/fermyon/developer/assets/111464359/d994f2ab-8113-40cf-861b-8d55e69737d5">